docs/api-markdown-documenter/custom-renderers.js

```diff
@@ -16,9 +16,9 @@ export function renderAlertNode(alertNode, writer, context) {
 	writer.ensureNewLine();
 
 	writer.writeLine(
-		`{{% callout ${alertNode.alertKind?.toLocaleLowerCase() ?? "note"} ${
+		`{{% callout ${alertNode.alertKind?.toLocaleLowerCase() ?? "note"} "${
```

Comment:
This was a bug. At some point, usages of the AlertNode must have been removed. Possibly in response to this? This fixes it, and allows us to leverage Hugo callouts as intended.